packages/core/src/components/List/List.tsx

```diff
@@ -52,6 +53,14 @@ export interface ListProps extends VibeComponentProps {
    */
   renderOnlyVisibleItems?: boolean;
   style?: CSSProperties;
+  /**
+   * ARIA role for the list. Default is "listbox".
+   */
+  role?: AriaRole;
+  /**
+   * ARIA role for the list items. Default is "option".
+   */
+  itemRole?: AriaRole;
```

Comment:
Will all the list items necessarily would have the same role if we wanna allow changing the role?

Reply:
Changed it to be a prop in the ListItem if you want to take another look
This is a legitimate case as there are several legit roles for a list